homework-submission/react-hyf/react-hyf-client/src/App.css

```diff
@@ -0,0 +1,121 @@
+body {
+  background-image: linear-gradient(rgba(255, 255, 255, 0.4), rgba(255, 255, 255, 1)),
+    url('https://i0.wp.com/www.thenextfact.com/wp-content/uploads/2017/10/reactjs.png?fit=1920%2C1080');
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: absolute;
```

Comment:
`position: absolute;` on the `body` is bad practice, remove the absolute from the body, and remove the `left: 40%` from the `.App` and your site will already be more responsive!